Flow.Launcher.Core/Configuration/Portable.cs

```diff
@@ -152,21 +154,17 @@ internal void IndicateDeletion(string filePathTodelete)
         public void PreStartCleanUpAfterPortabilityUpdate()
         {
             // Specify here so this method does not rely on other environment variables to initialise
-            var portableDataPath = Path.Combine(Directory.GetParent(Assembly.GetExecutingAssembly().Location.NonNull()).ToString(), "UserData");
-            var roamingDataPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "FlowLauncher");
-
-            bool DataLocationPortableDeleteRequired = false;
-            bool DataLocationRoamingDeleteRequired = false;
+            var portableDataDir = Path.Combine(Directory.GetParent(Assembly.GetExecutingAssembly().Location.NonNull()).ToString(), "UserData");
+            var roamingDataDir = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "FlowLauncher");
 
-            if ((roamingDataPath + "\\" + DataLocation.DeletionIndicatorFile).FileExits())
-                DataLocationRoamingDeleteRequired = true;
+            // Get full path to the .dead files for each case
+            var portableDataDeleteFilePath = Path.Combine(portableDataDir, DataLocation.DeletionIndicatorFile);
+            var roamingDataDeleteFilePath = Path.Combine(roamingDataDir, DataLocation.DeletionIndicatorFile);
 
-            if ((portableDataPath + "\\" + DataLocation.DeletionIndicatorFile).FileExits())
-                DataLocationPortableDeleteRequired = true;
-
-            if (DataLocationRoamingDeleteRequired)
+            // Should we switch from %AppData% to portable mode?
```

Comment:
What do you mean? This call is after the switch from portable to roaming or vice versa is done, to clean up either location leftover.

Reply:
You mean the comment is confusing, correct? I'll update it to clarify.

Reply:
yeah, just a little confusing, not sure what the comment is for. Is it something i can answer? (i wrote the code for it)